src/routes/group.route.ts

```diff
@@ -0,0 +1,27 @@
+import { Router } from 'express';
+import GroupsController from '@controllers/groupe.controller';
```

Comment:
change the filename to 
```suggestion
import GroupsController from '@controllers/group.controller';
```